api/cmd/main.go

```diff
@@ -101,3 +124,19 @@ func CorsHeader(c *gin.Context) {
 
 	c.Next()
 }
+
+func setupAzureBlobClient() (*azblob.Client, error) {
```

Comment:
Shouldn't the API fail fast if setting up the Azure Blob Client doesn't work (at least in production)?

Reply:
fixed